isb_lib/identifiers/datacite/__init__.py

```diff
@@ -35,7 +35,7 @@ def _validate_response(response) -> bool:
 
 
 def _post_to_datacite(
-    rsession: requests.session, post_data_str: typing.AnyStr, username: str, password: str
+    rsession: requests.Session, post_data_str: typing.AnyStr, username: str, password: str
```

Comment:
You'll see this same error fixed in a number of different places.  There is no such type as `requests.session` 🤦‍♂️

Reply:
oops